ui/src/app/app.tsx

```diff
@@ -175,7 +175,9 @@ export class App extends React.Component<
             const component = () => (
                 <>
                     <Helmet>
-                        <title>{extension.title} - Argo CD</title>
+                        <title>
+                            {extension.title} - {authSettings.uiTitle}
```

Comment:
This will only update the `title` of full page-view Extensions. To change the title of every page, you'll additionally have to add the `<title>` to the `<Helmet>` component at line 212 of this file. You can add a condition to only add this `<title>` if `uiTitle` is set.

To avoid issues when `authSettings.uiTitle` is not set, please change this to `{authSettings.uiTitle && authSettings.uiTitle !== '' ? authSettings.uiTitle : 'Argo CD'}`.

Reply:
Thanks, Yeah, I wanted to test a condition like that once I got it running,
I changed the 2 locations that have title, please review again when you can 🙏